arrow/arrow.py

```diff
@@ -80,12 +80,13 @@ class StochasticSystem(object):
     (and zero everywhere else). 
     '''
 
-    def __init__(self, stoichiometry, rates, random_seed=0):
+    def __init__(self, stoichiometry, rates, forms=None, random_seed=0):
```

Comment:
This needs a doc string, as I've forgotten what 'rates' is and don't know where to look. :)

Reply:
Are these stochastic rate constants, perhaps?  How do they play with the new 'forms'?  Are 'forms' like kinetic rate laws sans the outermost proportionality constant?